codeclimate-wrapper

```diff
@@ -18,32 +18,52 @@ EOF
   exit 1
 }
 
+docker_for_mac() {
+  test -S /var/tmp/docker.sock > /dev/null 2>&1
+}
+
+socket_exists() {
+  test -S $socket_path > /dev/null 2>&1
+}
+
+socket_path() {
```

Comment:
Also, sorry for the back and forth; we really appreciate you taking the time to work on this. Please let me know if you'd rather I commit what I'm thinking to this branch (or another) for your review instead.